### ISSUE TITLE
Issue with onenote imports

### ISSUE THREAD

**Issue by GuillaumeSeeker** [obsidianmd/obsidian-importer#316]
Hi, 
I was able to connect to my Microsoft account, it only shows 1 Notebook to import (i have 3 more on my Onenote app) , I see the 2 sections there nothing gets imported. The Error message appears: "Cannot read properties of undefined (reading 'value'",
Thank you in advance for your help

**Comment by adiyaojz** [comment]
me too

**Comment by PoppaColdOne** [comment]
Not sure if this will help - It took several minutes for all of my notebooks to show up once I was able to log in.  Also, I kept getting the same error as you guys, but was able to successfully import after many repeated attempts.  I found it helpful to close both my browser and Obsidian after failed attempts before retrying.  I also found it helpful to log into my microsoft account prior to trying to import so that the browser was already logged into microsoft on one tab.  Additionally, I turned of two factor authentication for this process.  It took many retries, but finally got it all done.  My main notebook is 450 MB.  Good luck!
